fhirr4/pom.xml

```diff
@@ -35,7 +35,7 @@
     <properties>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
-        <ballerina.version>2201.8.1</ballerina.version>
+        <ballerina.version>2201.9.1</ballerina.version>
         <gson.version>2.8.6</gson.version>
         <r4.version>5.0.0</r4.version>
         <parser.version>5.0.0</parser.version>
```

Comment:
Can we safely remove these as well?

Reply:
fixed in [8132297](https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/pull/326/commits/8132297b1d228e1302ee68e9b76f6fd613d09781)